esp/src/eclwatch/PackageMapPartsWidget.js

```diff
@@ -93,9 +93,8 @@ define([
                     var addPartInput = context.getFilter();
                     WsPackageMaps.AddPartToPackageMap({
                         request: {
-                            Target: addPartInput.AddPartsTarget,
-                            Process: addPartInput.Process,
-                            PackageMap: addPartInput.PackageMap,
+                            Target: context.params.target,
+                            PackageMap: context.params.packageMap.split('::')[1],
```

Comment:
the TARGET:: part of the PackageMap id is prepended internally.  There is support for global (unscopped) packagemap ids, although rarely used you should probably support IDs that don't start with TARGET::.